### PR TITLE
309 css framework

### PR DESCRIPTION
**Pull request by theabrad**
Set up the Sass for the project.

### CODE REVIEW

File: front-end/src/components/LearningEvent/LearningEvent.scss [hunk 1]
```diff
@@ -33,18 +32,6 @@
   font-family: "Source Sans Pro", sans-serif;
```
Comment (kdyer) [inline front-end/src/components/LearningEvent/LearningEvent.scss]: shouldn't this reference the main front from the base css?

File: front-end/src/components/TopicContentView/TopicContentView.scss [hunk 2]
```diff
@@ -51,28 +52,26 @@
   margin: 0 !important;
   padding: 1em 2em !important;
   border-bottom: 1px solid rgba(34, 36, 38, 0.15) !important;
-}
-
-.topic-content-event-grid:hover {
-  background-color: #f2f3f4 !important;
+  &:hover {
+    background-color: #f2f3f4 !important;
```
Comment (kdyer) [inline front-end/src/components/TopicContentView/TopicContentView.scss]: Does this have a corresponding variable from the style guide colors?

File: front-end/src/index.scss [hunk 3]
```diff
@@ -1,9 +1,12 @@
+@import "assets/style/base";
+
 body {
   margin: 0;
   padding: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  font-family: $main-font;
+  /* font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
```
Comment (kdyer) [inline front-end/src/index.scss]: Should we remove this comment?

File: front-end/src/components/TopicContentView/TopicContentView.scss (hunk 2 above)
Comment (mrcastle) [reply front-end/src/components/TopicContentView/TopicContentView.scss]: That was me, I couldn't find a color in the style guide to match so I pulled from mockup

File: front-end/src/components/LearningEvent/LearningEvent.scss (hunk 1 above)
Comment (theabrad) [reply front-end/src/components/LearningEvent/LearningEvent.scss]: whoops yeah

**Pull request merged**